src/sentry/options/defaults.py

```diff
@@ -113,6 +113,9 @@
 # The ratio of requests for which the new stackwalking method should be compared against the old one
 register("symbolicator.compare_stackwalking_methods_rate", default=0.0)
 
+# Ignore built-in sources during symbolication
+register("symbolicator.ignore_builtin_sources", default=False)
```

Comment:
I think this needs to be in a separate PR that we need to merge first.  Once merged we need to ask the right people to do the right thing (probably should use an ops request for this rather than pester the people who can do this directly).  Only then can we submit the rest.

Reply:
I might be mistaken (and can't find the jira task somehow?  still learning how that works) but I think the requirement is to specify a list of IDs which should be excluded.  So that ops can set the option to e.g. `sentry:electron` and if a project has both `sentry:electron` and `sentry:microsoft` enabled they'll still symbolicate with `sentry:microsoft` enabled.

There already are some options here registered as a "Sequence" so i guess we should be able to copy the syntax from those.

Reply:
Oh, you're right! I missed the part about filtering them out based on their ID, so I'll need to update this to reflect that, thank you. The matching logic required to make this work would mean it'd be a little difficult to use `sentry.killswitches` as-is, though. I'll leave a longer comment that goes over that.

Reply:
Actually, what's a little ambiguous to me is whether ops wants to kill _all_ built-in sources with one switch, or if they want to be able to globally kill specific built-in sources by providing a hit list. I interpreted it being the former, but perhaps they're interested in the latter functionality.